src/components/NewComment.jsx

```diff
@@ -0,0 +1,73 @@
+import { postComment } from "../utils/api";
+import { useState } from "react";
+
+export const NewComment = ({ article_id, loggedIn }) => {
+  const [newComment, setNewComment] = useState("");
+  const [postedComment, setPostedComment] = useState();
+  const [error, setError] = useState(false)
+  const [submitBuffer, setSubmitBuffer] = useState(false)
+
+  const postNewComment = (e) => {
+    e.preventDefault();
+    if (newComment !== "") {
+      setPostedComment({ body: newComment, author: "tickle122" });
+      setNewComment("");
+      postComment(article_id, newComment, "tickle122")
+      .catch(() => {
+        setError(true);
+      });
+      e.target[0].value = "";
+      setSubmitBuffer(true)
+      setTimeout(() => {setSubmitBuffer(false)}, 500)
+    }
+  };
+
+  const retryPostComment = () => {
+      setError(false)
+      postComment(article_id, postedComment.body, 'tickle122').catch(() => {
+        setError(true)
+    });
+  }
+
+  console.log(newComment);
+
+ if (loggedIn === true) {
+    return <div>
+      <form
+        onSubmit={(event) => {
+            if (loggedIn === true) {
+          postNewComment(event);
+            } 
+        }}
+        className="new-comment"
+      >
+        <textarea
+          className="new-comment-body"
+          onChange={(event) => {
+            setNewComment(event.target.value);
+          }}
```

Comment:
value attribute should ideally be here should be newComment if this is truly a controlled component